lib/datasource/packagist/__snapshots__/index.spec.ts.snap

```diff
@@ -554,10 +554,6 @@ Object {
       "gitRef": "0.0.1",
       "version": "0.0.1",
     },
-    Object {
-      "gitRef": "1.0.x-dev",
```

Comment:
Where does this version comes from?

Reply:
https://github.com/renovatebot/renovate/blob/60cbe8f69e38f7e7eff978e4c05071a72c579a9a/lib/datasource/packagist/index.spec.ts#L57